projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -400,6 +427,7 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
     this.clearForm();
     this.extractWorkflowParamsAlways(workflow);
     this.extractWorkflowParamsEnabled(workflow);
+    this.addIncrementalHarvestingFieldIfAvailable(workflow);
```

Comment:
Why not integrate the incremental checkbox in the method `extractWorkflowParamsAlways`? 
In your implementation you add an incremental harvesting control if needed. But if this method is called again (some reload or so) you will be adding a control again. Perhaps better to just always have the control but only enable/display it if necessary? Then it will behave just like the other OAI controls and things will be consistent and predictable.

Reply:
I've refactored it so it shows the incremental field by default and disables if it's unavailable, and the call is made from within `extractWorkflowParamsAlways`.